firefox-ios/Providers/Profile.swift

```diff
@@ -693,7 +693,36 @@ open class BrowserProfile: Profile {
             if !FileManager.default.fileExists(atPath: path) {
                 try? FileManager.default.createDirectory(at: url, withIntermediateDirectories: true)
             }
-            return try RemoteSettingsService(storageDir: path, config: RemoteSettingsConfig2(server: server))
+            let encoder = JSONEncoder()
+            let appInfo = BrowserKitInformation.shared
+            let uiDevice = UIDevice.current
+            let appContext = RemoteSettingsContext(
+                appId: "org.mozilla.ios.Firefox",
+                appVersion: AppInfo.appVersion,
+                appBuild: AppInfo.buildNumber,
+                channel: appInfo.buildChannel.rawValue,
+                deviceManufacturer: "Apple",
+                deviceModel: DeviceInfo.deviceModel(),
+                osVersion: uiDevice.systemVersion,
+                locale: Locale.current.identifier,
```

Comment:
This seems to be how other parts of the code does it, but I notice that `Locale` also has an `identifier` method that inputs a `Locale.IdentifierType`.  Should I be passing that in to make sure I get the right format?

Reply:
I think this is fine for now as that's what we use currently.